assets/js/pages/SettingsPage/SettingsPage.test.jsx

```diff
@@ -10,10 +10,25 @@ import {
   renderWithRouter,
 } from '@lib/test-utils';
 import { softwareUpdatesSettingsFactory } from '@lib/test-utils/factories/softwareUpdatesSettings';
+import { networkClient } from '@lib/network';
+import MockAdapter from 'axios-mock-adapter';
 
 import SettingsPage from './SettingsPage';
 
+const axiosMock = new MockAdapter(networkClient);
+
 describe('Settings Page', () => {
+  afterEach(() => {
```

Comment:
It would be nice to have a text to check if the api_key is displayed

Reply:
done